carl/envs/rna/data/download_and_build_eterna.py

```diff
@@ -0,0 +1,43 @@
+from tqdm import tqdm
+import requests
+
+
+def _download_dataset_from_http(url, download_path):
```

Comment:
TODO: mypy everything and this (add docstrings, return types)

Reply:
After conversation with Caro: only the CARL wrapper